.circleci/config.yml

```diff
@@ -173,7 +173,6 @@ jobs:
       - run:
           name: Setup Python environment
           command: |
-            pyenv global 3.5.2
```

Comment:
Ah, damn. I'm sorry. Now all the `pip` installs need to be `--user`. I thought I was making things simpler...

Reply:
Thanks. Yeah moving away from the machine images will make things simpler in the long tun.

Reply:
Will probably need it on L185, as well.

Reply:
Alternatively, we can consider `venv` (https://github.com/nipreps/dmriprep/blob/ae9a46710ce72b7e335c2a76d6cbb22f4f54150a/.circleci/config.yml#L250-L265)